airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx

```diff
@@ -64,6 +68,45 @@ type AuthContextApi = {
 
 export const AuthContext = React.createContext<AuthContextApi | null>(null);
 
+const getTempSignUpStorageKey = (currentUser: FbUser): string =>
+  `${currentUser.uid}/temp-signup-data`;
+const TempSignUpValuesProvider = {
+  get: async (
```

Comment:
:art: Wouldn't need to be `async` anymore. Same for the `save` method.

Reply:
Yeah, thought that there is no need to change the interface, but actually, as this method is only for temporary usage - let's get rid of `async` to make it clearer.